src/bridge/ex_data_util/ex_data_util.cc

```diff
@@ -104,7 +104,7 @@ Status AttachRsaKeyToOpenSslRsa(backing::RsaKey *rsa_key, RSA *rsa) {
   if (!RSA_set_ex_data(rsa, index, static_cast<void *>(rsa_key))) {
     return Status(StatusCode::kInternal, "RSA_set_ex_data failed");
   }
-  return Status();
+  return Status::kOk;
```

Comment:
It looks like absl is going to have a free function `OkStatus()`: https://github.com/abseil/abseil-cpp/blob/1995c6a3c2f9080160d9d8716504dc004e5e1ec0/absl/status/status.h#L362